### PR TITLE
Move delete button and improve poll results

### PR DESCRIPTION
**Pull request by elsmr**
- [x] remove delete button from poll details 
- [x] add delete button in poll list 
- [x] don't show error page when attempting a duplicate vote (message instead) 
- [x] random other stuff 
- [x]  show messages on delete

! need better styles for delete button

### CODE REVIEW

File: src/main/java/com/pollistics/controllers/PollController.java [hunk 1]
```diff
@@ -116,38 +121,22 @@ public String deletePoll(@PathVariable String pollId, HttpServletResponse respon
...
-			final int expiryTimeCookie = 2147483647; // maximum of int
-			final String cookiePath = "/";
-
-			cookie.setValue(value);
-			cookie.setMaxAge(expiryTimeCookie);
-			cookie.setPath(cookiePath);
-			response.addCookie(cookie);
+			return "redirect:/" + encodedSlug + "/results";
+		} else {			
+			response.addCookie(CookieBuilder.getVotedCookie(originalCookie, slug));
```
Comment (Haroenv) [inline src/main/java/com/pollistics/controllers/PollController.java:135]: Oh smart!

File: src/main/resources/templates/account/polls.twig [hunk 2]
```diff
@@ -4,11 +4,17 @@
 {% block description %}Login on Pollistics{% endblock %}
 
 {% block content %}
-<h1>Polls from {{ user.getUsername() }}</h1>
+<h1>Polls from {{ user.username }}</h1>
 <ul class="cta-list">
 {% for poll in polls %}
 	<li>
-		<a href="/{{ poll.getSlug() }}">{{ poll.getTitle() }}</a>
+		<div>
```
Comment (Haroenv) [inline src/main/resources/templates/account/polls.twig]: You can use the `<li>` as container too, right?

File: src/main/resources/templates/index.twig [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {% extends "layouts/main.twig" %}
 
-{% block pageTitle %}Pollistics Home{% endblock %}
```
Comment (Haroenv) [inline src/main/resources/templates/index.twig:3]: 👍

File: src/main/resources/templates/account/polls.twig (hunk 2 above)
Comment (elsmr) [reply src/main/resources/templates/account/polls.twig]: :+1:

**Review by Haroenv — APPROVED** [review]
Goed werk mijn jongen 

LGTM if tests 📗💚✳️

**Pull request merged**